syntaxes/graphql.connectors.json

```diff
@@ -0,0 +1,68 @@
+{
+  "scopeName": "inline.connectors.graphql",
+  "injectionSelector": "L:meta.embedded.block.directive.graphql -string -comment",
+  "patterns": [{ "include": "#connect-directive" }],
+  "repository": {
+    "connect-directive": {
+      "begin": "((@)\\s*(connect))",
+      "end": "(?=.)",
+      "applyEndPatternLast": 1,
+      "beginCaptures": {
+        "1": {
+          "name": "entity.name.function.directive.connect.graphql"
+        }
+      },
+      "patterns": [
+        { "include": "#connect-arguments" },
+        { "include": "source.graphql#graphql-comment" },
+        { "include": "source.graphql#literal-quasi-embedded" },
+        { "include": "source.graphql#graphql-skip-newlines" }
+      ]
+    },
+    "connect-arguments": {
+      "name": "meta.arguments.connect.graphql",
+      "begin": "\\s*(\\()",
+      "end": "\\s*(\\))",
+      "beginCaptures": {
+        "1": { "name": "meta.brace.round.directive.graphql" }
+      },
+      "endCaptures": {
+        "1": { "name": "meta.brace.round.directive.graphql" }
+      },
+      "patterns": [
+        { "include": "source.graphql#graphql-comment" },
+        { "include": "#connect-jsonselection-argument" },
+        { "include": "source.graphql#graphql-argument" },
+        { "include": "source.graphql#literal-quasi-embedded" }
+      ]
+    },
+    "connect-jsonselection-argument": {
+      "name": "meta.argument.connect.jsonselection.graphql",
+      "begin": "\\s*(selection)(?:\\s*(:))",
+      "end": "(?=\\s*(?:(?:([_A-Za-z][_0-9A-Za-z]*)\\s*(:))|\\)))|\\s*(,)",
+      "beginCaptures": {
+        "1": { "name": "variable.parameter.graphql" },
+        "2": { "name": "punctuation.colon.graphql" }
+      },
+      "endCaptures": {
+        "3": { "name": "punctuation.comma.graphql" }
+      },
+      "patterns": [
+        {
+          "contentName": "meta.embedded.block.jsonselection.graphql",
```

Comment:
Right now I'm stuck at the point where I'd need to inject the JSONSelection grammar into here.